### PR TITLE
🌱 e2e: dump cinder volumes

### PR DESCRIPTION
**Pull request by EmilienM**
**What this PR does / why we need it**:

This PR will help CI to collect the Cinder volumes that were created during the run.

### CODE REVIEW

File: test/e2e/shared/openstack.go [hunk 1]
```diff
@@ -349,6 +370,31 @@ func DumpOpenStackSecurityGroups(e2eCtx *E2EContext, filter groups.ListOpts) ([]
 	return groupsList, nil
 }
 
+// DumpOpenStackVolumes dumps all OpenStack volumes to a file in the artifact folder.
```
Comment (lentzi90) [inline test/e2e/shared/openstack.go:373]: nit:
```suggestion
// DumpOpenStackVolumes returns all OpenStack volumes based on the filter.
```

**Review by lentzi90 — COMMENTED** [review]
Nice!
/approve

File: test/e2e/shared/openstack.go (hunk 1 above)
Comment (EmilienM) [reply test/e2e/shared/openstack.go]: :+1:

**Pull request merged**